src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
redundant to reset margins for all selectors
better to reset for elements where it necessary